frontend/packages/shared/src/enums/common.ts

```diff
@@ -64,11 +64,13 @@ export enum HTTPStatusCode {
 
 export enum I18nNamespace {
   Accessibility = 'accessibility',
+  Clerk = 'clerk',
   Common = 'common',
   KoodistoCountries = 'koodistoCountries',
   KoodistoLanguages = 'koodistoLanguages',
   KoodistoRegions = 'koodistoRegions',
   Privacy = 'privacy',
+  Public = 'public',
```

Comment:
Mikä oli motivaatio näille uusille namespaceille?

Reply:
Ja oliko tässä siis ajatuksena että VKT:ssa Public+Clerk vastaa samaa kuin Translation AKRssä ja OTRssä?
Jos nää halutaan splitata kahteen namespaceen niin pitäisi samalla sitten tuo Translation poistaa?

Reply:
Kesän todo-listaa varten kun mietittiin, että voisi jakaa AKR:ssä ja OTR:ssä käännökset erllisten clerk ja public (tai vast. nimetyn) json-tiedoston alle, niin mietin että tuo voisi olla järkevä lähestymistapa nyt VKT:ssa kun sitä aletaan kehittämään. Samaan aikaan en näe, että kannattaisi tässä kohtaa ruveta samaa jumppaa tekemään AKR:n ja OTR:n puolella, jolloin olisi tuki tähän aiempaan translation.json:in käyttöön käännöksiä varten niissä kuten myös VKT:ssä publicin ja clerkin käyttöön.

Reply:
Ok. Ainakin se etu tuosta jakamisesta clerk- ja public-osioihin voisi olla että sitten olisi käännös-exceleissäkin selvempää mitkä käännökset koskee julkista UIta (eli tärkeitä) ja mitkä virkailijapuolta (vähemmän tärkeitä). Ja ei tosiaan ehkä kannata koske AKR&OTR käännöksiin samalla.

Reply:
Joo, tuota mietin myös että riittänee toistaiseksi, että luodaan VKT:ta varten asiakaspuolen käännöksistä exceliin tabi samoin kuin yleisistä käännöksistä (common). Ja sit toki myöhemmin selosteista. Jos nyt haluavat virkailijapuolen käännökset saada myös niin menee helposti omana tilauksenaan.